types/src/main/java/tech/pegasys/artemis/util/uint/UInt64.java

```diff
@@ -38,6 +39,10 @@ public long getValue() {
     return value;
   }
 
+  public int getIntValue() {
+    return (int) value;
+  }
```

Comment:
Ugh, scary thing :)
I would add the assertion `value >= 0 && value <= Integer.MAX_VALUE`